content/kubermatic/main/architecture/compatibility/supported-versions/_index.en.md

```diff
@@ -43,11 +43,17 @@ Upgrades from a previous Kubernetes version are generally supported whenever a v
 
 ## Provider Incompatibilities
 
+KKP has some incompatibilities with cloud providers, e.g. because their in-tree cloud provider
+implementation has been removed from upstream Kubernetes. For KKP 2.23.x, the following incompatibilities
+apply:
+## Provider Incompatibilities
+
```

Comment:
Not sure what happened here, looks like it duplicates a section that exists already.